apm/pricing.html

```diff
@@ -49,8 +49,8 @@ <h1 class="mb-0 display-4">Free</h1>
           environment
         </li>
       </ul>
-      <a class="btn btn-lg btn-primary w-100-sm-down onboarding-start-button px-4 py3" href="#" data-url="signup?external=yes">
-        Sign up for APM
+      <a class="btn btn-lg btn-primary w-100-sm-down onboarding-start-button px-4 py3 mr-3" href="#" data-url="signup?external=yes">
```

Comment:
Did you add this on purpose or just override my changes (I deleted these mr-3) by accident?
If it's here on purpose, I'd at least put it as mr-md-3, since this looks bad below md.

Reply:
Was by mistake. Pushed an update.